src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -382,6 +382,92 @@ class DemoLocalizationsDelegate extends LocalizationsDelegate<DemoLocalizations>
 }
 {% endprettify %}
 
+<a name="adding-language"></a>
+## Adding support for a new language
+
+An app that needs to support a language that's not included in
+[GlobalMaterialLocalizations]({{site.api}}/flutter/flutter_localizations/GlobalMaterialLocalizations-class.html)
+will have to do some extra work. It's not difficult, however to
+do the complete job one must provide about 70 translations
+("localizations") for words or phrases.
+
+As an example, we'll show how to add support for the Belarusan
+language.
+
+A new GlobalMaterialLocalizations subclass will define the
+localizations that the Material library depends on. Additionally one
+must create a LocalizationsDelegate subclass that essentially just
+constructs an instance of the new GlobalMaterialLocalizations
+subclass.
+
+Here's [the source code for a complete example](
+{{site.github}}/flutter/website/tree/master/examples/internationalization/add_language), 
+less the actual Belarusan translations, of an app that includes support for a new language.
+
+The locale-specific GlobalMaterialLocalizations subclass is called
+`BeMaterialLocalizations` and the LocalizationsDelegate subclass is
+`_BeMaterialLocalizationsDelegate`. The value of
+`BeMaterialLocalizations.delegate` is an instance of the delegate, and
+it's all that's needed by an app that uses these localizations.
+
+The delegate class includes basic date and number format
+localizations. All of the other localizations are defined by String
+valued property getters in BeMaterialLocalizations. Like this:
+
+{% prettify dart %}
+@override
+String get backButtonTooltip => r'Back';
+
+@override
+String get cancelButtonLabel => r'CANCEL';
+
+@override
+String get closeButtonLabel => r'CLOSE';
+
+// etc..
+{% endprettify %}
+
+These are the English translations of course. To complete the job one must change the return value of each getter to an appropriate Belarusan string.
+
+The getters return "raw" Dart strings that have an r prefix, like
+`r'About $applicationName'`, because sometimes the strings contain
+`$variables` that are expanded by methods with parameters, like
```

Comment:
`$variables` seems questionable, since it's literal and yet not. Maybe this (which you could probably improve):

variables with a `$` prefixes. Methods replace these variables with parameter values,
as in the following example: